auacm/app/modules/competition_manager/views.py

```diff
@@ -158,10 +158,16 @@ def get_competition_data(cid):
     scoreboard = list()
 
     team_users = dict()
+    team_display_names = dict()
+
     for user in comp_users:
         if not user.team in team_users:
             team_users[user.team] = list()
+            team_display_names[user.team] = list()
         team_users[user.team].append(user.username)
+        team_display_names[user.team].append(database.session.query(User)
+                                            .filter(User.username == user.username)
+                                            .first().display)
```

Comment:
Yeah, it looks like there's no caching, and [queries are expensive](https://gist.github.com/BrandonLMorris/3e2ae042911026a9f04ec40f33dbfb98). What do you think @mcprice30 ?